lib/mayaUsd/nodes/proxyShapeBase.cpp

```diff
@@ -584,7 +585,7 @@ MayaUsdProxyShapeBase::computeInStageDataCached(MDataBlock& dataBlock)
             }
             else {
                 // Create a new stage in memory with an anonymous root layer.
-                usdStage = UsdStage::CreateInMemory("", loadSet);
+                usdStage = UsdStage::CreateInMemory(kanonymousLayerName, loadSet);
```

Comment:
@seando-adsk  Setting identifier passed to **CreateInMemory** routine automatically sets the anonymous name.

See here:
https://github.com/PixarAnimationStudios/USD/blob/ebac0a8b6703f4fa1c27115f1f013bb9819662f4/pxr/usd/usd/stage.cpp#L777